parm/fv3lam_rrfs.xml

```diff
@@ -593,7 +592,6 @@
        <param>
        <shortname>LFTX_ON_ISOBARIC_SFC_500-1000hpa</shortname>
        <pname>LFTX</pname>
```

Comment:
@AndrewBenjamin-NOAA Do you think the change of 4LFTX number from 193 to 11 is needed?